src/xlsx/conversion/figure_conversion.ts

```diff
@@ -33,20 +34,44 @@ function convertFigure(
   const width = x2 - x1;
   const height = y2 - y1;
 
-  const chartData = convertChartData(figure.data);
-  if (!chartData) return undefined;
-  return {
-    id: id,
-    x: x1,
-    y: y1,
-    width: width,
-    height: height,
-    tag: "chart",
-    data: convertChartData(figure.data),
-  };
+  if (isChartData(figure.data)) {
+    return {
+      id: id,
+      x: x1,
+      y: y1,
+      width: width,
+      height: height,
+      tag: "chart",
+      data: convertChartData(figure.data),
+    };
+  } else if (isImageData(figure.data)) {
+    const imageData = figure.data;
+    if (!imageData.path) return undefined;
+    const imageHeight = width / imageData.aspectRatio;
```

Comment:
why using the original aspect ratio ? we should respect the intention of the user and use the figure dimensions don't we ?

Reply:
> why using the original aspect ratio ? we should respect the intention of the user and use the figure dimensions don't we ?

@rrahir Currently the figure dimensions cannot be accurately converted (we discussed it before, because of `twoCellAnchor` from Excel and confusing conversion of row/col sizes) so in any way we are not able to respect the intention of the user for now 😂 

IMO it's more often that the user stretch the image in its original aspect ratio, so I made it this way. 

(I had the discussion with FRGI two months ago about this aspect ratio problem, see log notes here: https://www.odoo.com/web#id=3092986&cids=1&menu_id=4720&action=333&active_id=2328&model=project.task&view_type=form)